internal/config/attest.go

```diff
@@ -1,19 +1,29 @@
 package config
 
 import (
+	"fmt"
+	"github.com/mitchellh/go-homedir"
```

Comment:
:sadpanda: we need a commit hook for this!

Reply:
I haven't gotten gosimports yet into syft, in which case, yeah, this'll be a thing of the past